### ISSUE TITLE
Refactor prompt construction code

### ISSUE THREAD

**Issue by jpodivin** [fedora-copr/logdetective#93]
Prompts are constructed out of built-in strings, often used as f-string. I would prefer them to be loaded from a file.
Ideally yaml, or other similar, easily parsable format.

**Comment by TomasTomecek** [comment]
Agreed! Changing prompts without editing code sounds like a great idea - that will make experimenting much easier.

**Comment by abadger** [comment]
This appears to be implemented in https://github.com/fedora-copr/logdetective/pull/184 and can be closed.
